pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -188,8 +192,8 @@ var _ = Describe("Pod Network", func() {
 		mockNetwork.EXPECT().AddrAdd(bridgeTest, masqueradeGwAddr).Return(nil)
 		mockNetwork.EXPECT().AddrAdd(bridgeTest, masqueradeIpv6GwAddr).Return(nil)
 		mockNetwork.EXPECT().StartDHCP(masqueradeTestNic, masqueradeGwAddr, api.DefaultBridgeName, nil)
-		mockNetwork.EXPECT().GetHostAndGwAddressesFromCIDR(api.DefaultVMCIDR).Return("10.0.2.1/30", "10.0.2.2/30", nil)
-		mockNetwork.EXPECT().GetHostAndGwAddressesFromCIDR(api.DefaultVMIpv6CIDR).Return("fd10:0:2::1/120", "fd10:0:2::2/120", nil)
+		mockNetwork.EXPECT().GetHostAndGwAddressesFromCIDR(ipv4CidrStr).Return(masqueradeGwStr, masqueradeVmStr, nil)
```

Comment:
I'm sorry, but I still don't understand why do you need a new variable instead of using the default cidr const? Keeping the default const here emphasizes the fact no user defined cidr was specified.

Reply:
In my opinion its easier to understand and to maintain these kind of tests by keeping all configurations in one place, in this case the configuration is the IP addresses.
The fact that we take the default CIDRs from `api` shouldn't  matter, if there would be any change and the default CIDR will not be in the `api` anymore (which I think it shouldn't be there but that for another day (; ) , we would change the variable that represent it w/o touching the test itself cussing potential bugs.
This is also my main reason to switch from hard-coded addresses to representative variables.

Reply:
There is no need to discuss this in two/four places, closing this conversation in favor of the ones opened on the variables themselves (line 111).